cql3/relation.hh

```diff
@@ -220,6 +222,12 @@ public:
     virtual ::shared_ptr<restrictions::restriction> new_contains_restriction(database& db, schema_ptr schema,
         ::shared_ptr<variable_specifications> bound_names, bool isKey) = 0;
 
+    /**
+     * Creates a new LIKE restriction instance.
+     */
+    virtual ::shared_ptr<restrictions::restriction> new_LIKE_restriction(database& db, schema_ptr schema,
+        ::shared_ptr<variable_specifications> bound_names) = 0;
```

Comment:
Broken line needs extra indent.

Reply:
This is identical to the old code above it.